packages/env/lib/env.js

```diff
@@ -469,28 +468,34 @@ async function configureWordPress( environment, config ) {
 		if ( typeof value !== 'string' ) {
 			command.push( '--raw' );
 		}
-		await dockerCompose.run(
-			environment === 'development' ? 'cli' : 'tests-cli',
-			command,
-			options
-		);
+		await dockerCompose.run( containerName, command, options );
 	}
 
 	// Activate all plugins.
-	for ( const pluginSource of config.pluginSources ) {
-		await dockerCompose.run(
-			environment === 'development' ? 'cli' : 'tests-cli',
-			`wp plugin activate ${ pluginSource.basename }`,
-			options
-		);
+	await dockerCompose.run(
+		containerName,
+		'wp plugin activate --all',
+		options
+	);
+
+	// Get the list of themes available to WordPress.
+	const themeOutput = await dockerCompose.run(
+		containerName,
+		'wp theme list --fields=name',
+		options
+	);
+
+	// In case of an error getting the themes, do nothing.
+	if ( themeOutput.exitCode !== 0 ) {
+		return;
 	}
 
-	// Activate the first theme.
-	const [ themeSource ] = config.themeSources;
-	if ( themeSource ) {
+	// Index 0 contains "name" (the header of the output), so use next index.
+	const [ , firstTheme ] = themeOutput.out.split( '\n' );
```

Comment:
Are we guaranteed that the first theme listed by `wp theme list` is the first theme specified in `.wp-env.json`'s `"themes"` array? It may be that WP-CLI lists themes alphabetically.

Reply:
good point. I changed it to still activate the first theme in the array if we are using the array format.

I wonder, do we want this functionality? If the user activates a different theme while using wp-env, wouldn't this overwrite their choice every time they restart the container?

Reply:
That's a good question. I think it makes sense to have `wp-env` activate a theme because it mirrors what it does for plugins. But now you have me questioning myself... 😀

Reply:
Yeah.. I mean, either are easy enough to activate internally. I guess the difference is that if you list all plugins, you probably want them all to be running most of the time. But with themes, only one can run at a time, so why not let the user choose in wp-admin? (I bet we'll get a "bug report" about the behavior at some point ;))